.github/actions/check-pr-title/action.yml

```diff
@@ -0,0 +1,23 @@
+---
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: "Check PR Title"
+description: "Checks if the pull request title follows Conventional Commits format."
+inputs:
+  pr_title:
+    description: "The title of the pull request."
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        echo "Checking PR title: ${{ inputs.pr_title }}"
```

Comment:
`${{ inputs.pr_title}}` should be an environment variable.

The reason being, user inputs are not santized, and can harbor commands which get evaluated at runtime.  For example, if `inputs.pr_title` is set to `$(rm -rf /*)`, bad things happen.